### DISCUSSION TITLE
请问EN国际服自动基建功能做得怎样了？

### DISCUSSION THREAD

**Discussion by KiriyaXX** [MaaAssistantArknights/MaaAssistantArknights#3208] [Q&A] [answered]
找到了一串5个月前的讨论
https://github.com/MaaAssistantArknights/MaaAssistantArknights/discussions/1212
现在还需要帮忙截图吗？

**Comment by ABA2396** [comment, marked as answer]
[#3162](https://github.com/MaaAssistantArknights/MaaAssistantArknights/pull/3162) 已经基本解决了吧，可以试试使用的时候哪里卡住了

**Reply by KiriyaXX** [reply]
doc 里面没写，我还以为不能用呢
好的我去试试
